Automattic/wp-calypso
# Site Settings: Run codemods on delete-site and delete-site-warning-dialog

This PR contains updates from the following codemods I've ran on `site-settings/delete-site` and `site-settings/delete-site-warning-dialog`:

* commonjs-imports-hoist
* commonjs-imports
* commonjs-exports
* i18n-mixin
* react-create-class
* react-prop-types

Seems like the code was up to date there, the only update necessary was the prop-types package in  the corresponding components.

To test:
* Checkout this branch
* Go to http://calypso.localhost:3000/settings/general/:site, where :site is one of your WordPress.com sites with a Paid plan.
* Click on the Delete Site button in the bottom and verify you can see the warning dialog, without any warnings.
* Go to http://calypso.localhost:3000/settings/delete-site/:site, where :site is one of your WordPress.com sites with a Free plan.
* Verify you can see the cards properly and you see no warnings.
